exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/utils/slc_ui.ipynb

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #134.        if not clone_slc_repo(conf):
</pre><p>You have already checked that clone_slc_repo(conf) == False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>You are right. Removed the second check.</p>